core/frontend/src/components/parameter-editor/ServoFunctionActuatorEditor.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div class="d-flex align-center">
+    <v-row>
+      <v-col cols="6">
+        <span class="text-h6">Increment</span>
+      </v-col>
+      <v-col cols="6">
+        <inline-parameter-editor
+          :label="actuator_inc_param?.name"
+          :param="actuator_inc_param"
+        />
+      </v-col>
+      <v-row>
+        <v-col cols="12">
+          <v-alert
+            v-if="!has_joystick_function_configured"
+            type="error"
```

Comment:
I'm unsure whether this should be an error or just a warning, since someone may just want to control an Actuator programatically (e.g. with a Cockpit slider, or some Actions).

That said, I suppose _right now_ it's not possible to do that, so we could make it an error and update it once `DO_SET_ACTUATOR` support is added?